jsdelivr-npm-importmap.js

```diff
@@ -1,73 +1,43 @@
-    "@marlowe.io/language-core-v1/semantics":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/language-core-v1@0.2.0-beta/dist/bundled/esm/semantics.js",
-    "@marlowe.io/language-core-v1/version":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/language-core-v1@0.2.0-beta/dist/bundled/esm/version.js",
-    "@marlowe.io/language-examples":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/language-examples@0.2.0-beta/dist/bundled/esm/language-examples.js",
-    "@marlowe.io/language-specification-client":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/language-specification-client@0.2.0-beta/dist/bundled/esm/language-specification-client.js",
-    "@marlowe.io/token-metadata-client":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/token-metadata-client@0.2.0-beta/dist/bundled/esm/token-metadata-client.js",
-    "@marlowe.io/wallet":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/wallet@0.2.0-beta/dist/bundled/esm/wallet.js",
-    "@marlowe.io/wallet/api":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/wallet@0.2.0-beta/dist/bundled/esm/api.js",
-    "@marlowe.io/wallet/browser":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/wallet@0.2.0-beta/dist/bundled/esm/browser.js",
-    "@marlowe.io/wallet/nodejs":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/wallet@0.2.0-beta/dist/bundled/esm/nodejs.js",
-    "@marlowe.io/runtime-rest-client":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-rest-client@0.2.0-beta/dist/bundled/esm/runtime-rest-client.js",
-    "@marlowe.io/runtime-rest-client/transaction":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-rest-client@0.2.0-beta/dist/bundled/esm/transaction.js",
-    "@marlowe.io/runtime-rest-client/withdrawal":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-rest-client@0.2.0-beta/dist/bundled/esm/withdrawal.js",
-    "@marlowe.io/runtime-core":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-core@0.2.0-beta/dist/bundled/esm/runtime-core.js",
-    "@marlowe.io/runtime-lifecycle":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-lifecycle@0.2.0-beta/dist/bundled/esm/runtime-lifecycle.js",
-    "@marlowe.io/runtime-lifecycle/api":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-lifecycle@0.2.0-beta/dist/bundled/esm/api.js",
-    "@marlowe.io/runtime-lifecycle/browser":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-lifecycle@0.2.0-beta/dist/bundled/esm/browser.js",
-    "@marlowe.io/runtime-lifecycle/generic":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/runtime-lifecycle@0.2.0-beta/dist/bundled/esm/generic.js",
-    "@marlowe.io/marlowe-object":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/marlowe-object@0.2.0-beta/dist/bundled/esm/marlowe-object.js",
-    "@marlowe.io/marlowe-object/guards":
-      "https://cdn.jsdelivr.net/npm/@marlowe.io/marlowe-object@0.2.0-beta/dist/bundled/esm/guards.js",
-    "lucid-cardano": "https://unpkg.com/lucid-cardano@0.10.7/web/mod.js",
-  },
+  "imports": {
```

Comment:
this amount of change indicates me that you have not done a treefmt. We need to add that as a CI check. I created this Issue to remind ourselves 

https://github.com/input-output-hk/marlowe-ts-sdk/issues/115